### ISSUE TITLE
Add debug instructions

### ISSUE THREAD

**Issue by caseyyee** [caseyyee/unity-webvr-export#63]
Enabling debug builds should make it easier to see errors in Unity script in the web console.   Will need to test to see if this works.

**Comment by cvan** [comment]
From here?

![image](https://user-images.githubusercontent.com/203725/34810640-a4e7234c-f66a-11e7-8dcd-784bae55ed7f.png)

Is there a way to define a setting to enable `Development Builds` by default from the Unity asset package or project settings?

**Comment by caseyyee** [comment]
Yup, I think so.   Just a tip made to me, but I haven't tried it out yet.

Not sure about where you might be able to toggle this from other than the platform settings.

**Comment by caseyyee** [comment]
https://docs.unity3d.com/Manual/webgl-building.html
